examples/experimental-dev/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "experimental-dev",
-  "version": "5.0.0-rc.8",
```

Comment:
usually this get auto bumped during release, with the version being 0.0.0 isn't it breaking anything. Can you try running an experimental release on this branch to see if everything works ?

Reply:
In the couple of releases I did, those weren't upgraded automatically, but I felt the need to fix that manually because they were referencing previous v5 RC versions.

I just did an experimental release (locally, not github actions) and the result is odd, it didn't error but only upgraded the package.json for lerna.json and 6 package.jsons 🤔

Reply:
Releasing via github actions seems to work as expected

Reply:
Ok that's odd, Let's merge it and try it in real condition with the next v5 rc, worst case is we revert and do another rc